pandas/io/sas/sas7bdat.py

```diff
@@ -687,6 +687,26 @@ def _read_next_page(self):
 
         return False
 
+    @staticmethod
```

Comment:
rather than a static method can you just make this module level

Reply:
OK